src/Maestro/Maestro.Data/Models/Build.cs

```diff
@@ -35,6 +35,18 @@ static Build()
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public int Id { get; set; }
 
+        public string Account { get; set; }
+
+        public string Project { get; set; }
+
+        public string AzDoRepository { get; set; }
+
+        public string AzDoBranch { get; set; }
+
+        public int AzDoBuildId { get; set; }
+
+        public int SourceBuildDefinitionId { get; set; }
```

Comment:
What is this one?  We might want some comments, since we have 3 "build ids" in this thing now to explain which ones are set when for what purposes.

Reply:
ID of the build definition that originated the build as per your doc :-). I'll add the comments.

Reply:
Oooo. I'd just call is AzureDevOpsBuildDefinitionId. "SourceBuild" is another thing, and will probably also end up in the DB before long. :-)

Reply:
Actually, I think these should be an "Owned Type" thing. https://docs.microsoft.com/en-us/ef/core/modeling/owned-entities

So we have an AzureDevOpsBuildInfo object as a property of the build, and it maps to a bunch of columns in the Builds table, but it doesn't clutter the build object with properties.

Reply:
What do you guys think of also renaming some existing fields, like `BuildNumber` to `AzureDevOpsBuildNumber` and `Repository` to `GithubRepository`, etc ? And then spliting these fields in two owned type classes `AzureDevOpsBuildInfo` and `GitHubBuildInfo` (if that makes sense..).

Reply:
I'm okay with that, we would have to make sure the migration is created to rename the columns properly though for the existing fields.

Reply:
Using "owned type" entities make the column names quite long, like `AzureDevOpsBuildInfo_AzureDevOpsRepository` should I look to shorten the names (i.e., just `AzureDevOpsRepository`) or let it the way it is?

Reply:
Once it's contained, the inner thing can just say Respository, so it's just AzureDevOps.Respository.  Basically just add a dot in the middle of the existing names. :-)